src/main/java/ArrayMinHeap.java

```diff
@@ -47,6 +47,22 @@ private void bubbleUp(int index) {
         }
     }
 
+    private void bubbleDown(int index) {
```

Comment:
Format is **still** inconsistent. Enable format on save.

Reply:
it is. It's being weird locally for some reason 

![image](https://user-images.githubusercontent.com/1328048/202491519-8b446ada-c715-458c-99a5-f5a7835485e0.png)